app/src/pipelines/generate_referrals/pipeline_wrapper.py

```diff
@@ -65,16 +67,15 @@ def run_api(self, query: str, prompt_version_id: str = "") -> dict:
             prompt_template = haystack_utils.get_phoenix_prompt(
                 "generate_referrals", prompt_version_id
             )
-            logger.info("Overriding the prompt_version with %s", prompt_version_id)
-            # NOTE: the supports are loaded in by the LoadSupports component in setup()
```

Comment:
Moved this log message to `app/src/common/phoenix_utils.py`. Also the message is not entirely accurate since we're not overriding every time.